qa/tasks/ceph_manager.py

```diff
@@ -1362,15 +1362,18 @@ def raw_cluster_cmd(self, *args, **kwargs):
         """
         Start ceph on a raw cluster.  Return count
         """
-        stdout = kwargs.pop('stdout', StringIO())
-        p = self.run_cluster_cmd(args=args, stdout=stdout, **kwargs)
-        return p.stdout.getvalue()
+        if not kwargs:
+            kwargs['args'] = args
```

Comment:
Looking at the history for this method, notably c0907f99e87898f7e0afd47f4ed143b20a477cb7, `kwargs['args']` has always been ignored. Why would we want to change that?

Reply:
Because accepting kwargs but ignoring command arguments is buggy. The tracker ticekts linked in the commits and on this PR demonstrate the issue with an example. PTAL -
https://tracker.ceph.com/issues/49495
https://tracker.ceph.com/issues/49486

Reply:
Ahh! You must have had following case in your mind. I've covered following case in the upcoming push -
```
            cls.mgr_cluster.mon_manager.raw_cluster_cmd_result(*cmd, stdin=json.dumps(
                cls.ORCHESTRATOR_TEST_DATA
            ))
```

Reply:
@batrick